src/Analysis/Ast/Impl/Microsoft.Python.Analysis.csproj

```diff
@@ -9,7 +9,7 @@
       1701, 1702 - "You may need to supply assembly policy"
     -->
     <NoWarn>1701;1702;$(NoWarn)</NoWarn>
-    <LangVersion>7.2</LangVersion>
```

Comment:
Does this language version need to stay? I think we had put it there for a reason, but I can't recall exactly.

Reply:
asked dotnet people and they said most of people just use latest C# lang version with .netstandard 2.0 and just make sure not to use some language features that rely on .net core 3.0 such as default interface method.